includes/rest-api/endpoints/class-wp-rest-abilities-list-controller.php

```diff
@@ -128,13 +128,13 @@ public function get_items( $request ) {
 		if ( $page > 1 ) {
 			$prev_page = $page - 1;
 			$prev_link = add_query_arg( 'page', $prev_page, $base );
-			$response->add_link( 'prev', $prev_link );
+			$response->link_header( 'prev', $prev_link );
```

Comment:
This is a breaking fix, but I also doubt anyone was using pagination yet.

We use link_header() instead of add_link() to keep the response as a pure array.

When using add_link(), it would add "_links" to the response body, converting it from an array / to a JSON object like {"0": {...}, "_links": {...}}.

Reply:
Created https://github.com/WordPress/abilities-api/pull/81 just to make sure we get this in and don't cause issues for ourselves.